### PR TITLE
Capitalize word on webPhar page

### PR DESCRIPTION
**Pull request by zors1**
Capitalized a word at the beginning of a sentence on https://www.php.net/manual/en/phar.webphar

### CODE REVIEW

**Review by saundefined — APPROVED** [review]
Thank you!

File: reference/phar/Phar/webPhar.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 <refentry xml:id="phar.webphar" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink">
  <refnamediv>
   <refname>Phar::webPhar</refname>
-  <refpurpose>mapPhar for web-based phars. front controller for web applications</refpurpose>
+  <refpurpose>mapPhar for web-based phars. Front controller for web applications</refpurpose>
```
Comment (tiffany-taylor) [inline reference/phar/Phar/webPhar.xml:6]: This should have a period at the end

Comment (tiffany-taylor) [inline reference/phar/Phar/webPhar.xml:6]: This should have a period at the end as well.

Comment (zors1) [reply reference/phar/Phar/webPhar.xml:6]: Right, my bad. I'll submit a new PR.

Comment (tiffany-taylor) [reply reference/phar/Phar/webPhar.xml:6]: Thanks!

**Pull request merged**